common/discordutil.js

```diff
@@ -59,7 +60,20 @@ module.exports = {
         console.log(channelID, classCode);
 
         file.ids[channelID] = classCode;
-        this.writeToFile();
+        this.writeToFile(pathToJson, file);
+        return true;
+    },
+    addPreviewChannel(channelID, message){
```

Comment:
do you think there is a case to add a `removePreviewChannel` and respective command?